Traceback (most recent call last):
  File "/home/rtcalumby/adam/luciano/LifeCLEFPlant2022/DeepCluster/main_FGDCC_v2.py", line 16, in <module>
    from engine_FGDCCv2 import main as app_main
  File "/home/rtcalumby/adam/luciano/LifeCLEFPlant2022/DeepCluster/engine_FGDCCv2.py", line 47, in <module>
    from src.datasets.FineTuningDataset import make_GenericDataset
  File "/home/rtcalumby/adam/luciano/LifeCLEFPlant2022/DeepCluster/src/datasets/FineTuningDataset.py", line 11, in <module>
    import torchvision
  File "/home/rtcalumby/miniconda3/envs/py382/lib/python3.12/site-packages/torchvision/__init__.py", line 6, in <module>
    from torchvision import _meta_registrations, datasets, io, models, ops, transforms, utils
  File "/home/rtcalumby/miniconda3/envs/py382/lib/python3.12/site-packages/torchvision/_meta_registrations.py", line 8, in <module>
    import torchvision.extension  # noqa: F401
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/rtcalumby/miniconda3/envs/py382/lib/python3.12/site-packages/torchvision/extension.py", line 92, in <module>
    _check_cuda_version()
  File "/home/rtcalumby/miniconda3/envs/py382/lib/python3.12/site-packages/torchvision/extension.py", line 78, in _check_cuda_version
    raise RuntimeError(
RuntimeError: Detected that PyTorch and torchvision were compiled with different CUDA major versions. PyTorch has CUDA Version=12.1 and torchvision has CUDA Version=11.8. Please reinstall the torchvision that matches your PyTorch install.
